.github/workflows/ci-workflow.yml

```diff
@@ -49,19 +49,6 @@ jobs:
       - name: Run all tests
         run: |
           make test
-      - name: Generate codecov report
-        uses: codecov/codecov-action@v2
-        with:
-          files: ./test_results/latest/testcoverage.out # optional
-          flags: unittests # optional
-          name: codecov-umbrella # optional
-          fail_ci_if_error: true # optional (default = false)
-          verbose: true # optional (default = false)
-      - name: Archive code coverage results
```

Comment:
I think this can stay

Reply:
I was thinking of removing this portion instead? https://github.com/CMSgov/bcda-app/pull/933/files#diff-e42b2b9794d0333e41c8e218ece5897ea7395746b5e07d08fb85b53e36d15043L71-L74 

If we remove the codecov report generation but keep in the artifact, would it fail when it searches the path for the `testcoverage.out`?

Reply:
That's what I'm thinking; it's going to look for code-coverage-report and then fail

Reply:
go has built-in test coverage tools, so the file still gets generated, from what I can tell.